root/etc/cont-init.d/50-config

```diff
@@ -222,31 +222,35 @@ sed -i 's|^certbot_dns_transip:||g' /config/dns-conf/transip.ini
 
 # setting the validation method to use
 if [ "$VALIDATION" = "dns" ]; then
-    if [ "$DNSPLUGIN" = "route53" ]; then
-        if [ -n "$PROPAGATION" ];then PROPAGATIONPARAM="--dns-${DNSPLUGIN}-propagation-seconds ${PROPAGATION}"; fi
-        PREFCHAL="--dns-${DNSPLUGIN} ${PROPAGATIONPARAM}"
+    if [[ "$DNSPLUGIN" =~ ^(azure)$ ]]; then
+        if [ -n "$PROPAGATION" ];then echo "Azure dns plugin does not support setting propagation time"; fi
```

Comment:
Is this relevant to this PR?

Reply:
From PR description:

Alphabetized the listing of dns validation check statements within 50-config.